tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "moduleResolution": "node",
         "sourceMap": false,
         "jsx": "react",
+        "skipLibCheck": true,
```

Comment:
Why was this necessary?

Reply:
yeah. this is 100% the reason for this.   the TS extension is causing the issue.  see other issue I opened up against the forked repo.
https://github.com/microsoft/AzureDevOps-WSJF-Extension/issues/16

Reply:
I would love an option to NOT do this..however I don't see an alternative.